packages/@sfcc-bff/basketapi/src/api/schema/basketTypeDef.ts

```diff
@@ -44,7 +44,29 @@ export const basketTypeDef = gql`
         price: Float!
         quantity: Int!
         itemId: String!
-        image: String
+        imageURL: String
```

Comment:
if we put these in the product api typedefs for product, then we can simply use that typedef here.

Reply:
same as above.

Reply:
typedefs all get aggregated into one big string in the end before we start the apollo server, so it can work there.

Reply:
same as above.